packages/analytics-core/src/timeline.ts

```diff
@@ -93,3 +93,13 @@ export const apply = async () => {
 
   return;
 };
+
+export const flush = async (config: Config) => {
```

Comment:
my understanding of flush is that it is typically called before server exits, and meant to send all unset events. i think this would only send events that are in the destination plugin's queue, which may not include all unsent events. Other unsent events are in [timeline queue](https://github.com/amplitude/Amplitude-TypeScript/blob/92325bc34cf0143f5f33ec4b0afd3e2d148c3d38/packages/analytics-core/src/timeline.ts#L15) (pending to be applied with before and enrichment plugins) and [delayed retryable events queue](https://github.com/amplitude/Amplitude-TypeScript/blob/f353367b8b264f86b6ea15b15f30385f8d5b8ad5/packages/analytics-core/src/plugins/destination.ts#L80) in amplitude destination plugin.